derp/xdp/xdp.c

```diff
@@ -14,6 +14,10 @@ struct config {
 	// the context of the data. cilium/ebpf uses native endian encoding for map
 	// encoding even if we use big endian types here, e.g. __be16.
 	__u16 dst_port;
+	// If drop_stun is set to a nonzero value all UDP packets destined to
+	// dst_port will be dropped. This is useful for shedding home client load
+	// during maintenance.
+	__u16 drop_stun;
```

Comment:
Any particular reason to use __u16 vs any other size?

Reply:
__u8 results in a padding byte, and I preferred to avoid that